polls/views.py

```diff
@@ -1,11 +1,34 @@
-from django.http import HttpResponse
+from django.views import View
 from django.shortcuts import render
-from django.template import loader
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.views.generic import TemplateView
+from django.utils import timezone
 
-# Create your views here.
+class IndexView(TemplateView):
+    template_name = "polls/index.html"
 
+class AuthFlowView(LoginRequiredMixin, View):
+    template_name = "polls/auth_flow.html"
+    min_time = 900
+    max_time = 4500
```

Comment:
Can you give these better variable names?
like `nminutes_before_should_rest`
and `nminutes_before_can_continue`